proctor-consumer/src/main/java/com/indeed/proctor/consumer/ProctorConsumerUtils.java

```diff
@@ -188,8 +191,8 @@ private static Cookie doCreateForcedGroupsCookie(
      */
     @Deprecated
     @Nonnull
-    public static Map<String, Integer> parseForcedGroups(@Nonnull final HttpServletRequest request) {
-        return parseForcedGroupsOptions(request).getForceGroups();
+    public static Map<String, Integer> parseForcedGroups(@Nonnull final HttpServletRequest request, Set<String> forcePayloadTests) {
```

Comment:
As same as the above comment, users of this method will get a compile error.

In this case, I suggest keeping public static Map<String, Integer> parseForceGroupsList(@Nullable final String payload) and don't add the new signature until we find it's necessary to add.